.github/actions/check-for-CLA/index.js

```diff
@@ -103,7 +103,7 @@ const checkIfUserHasSignedAnyCLA = async () => {
 
 const getCommentBody = (hasSignedCLA, errorFoundOnCLACheck) => {
   const commentTemplate = fs.readFileSync(
-    "./.github/actions/check-for-CLA/templates/pullRequestComment.hbs",
+    "./templates/pullRequestComment.hbs",
```

Comment:
Would this be more robust to actually build the file path it expects?

```js
join(__dirname, './templates/pullREquestComment.hbs')
```

Reply:
Yep, done!